src/libpmemfile-posix/dir.c

```diff
@@ -298,6 +298,7 @@ vinode_add_dirent(PMEMfilepool *pfp,
  * vinode_new_dir -- creates new directory relative to parent
  *
  * Note: caller must hold WRITE lock on parent.
+ * Must be called in transaction.
```

Comment:
Didn't we talk about some using some naming convention as well for this?
Where adding a ```_tx``` means some requirement.
Perhaps we should agree on a convention at least among us (NVML & pmemfile & pmemkv & stuff done inside Intel)